### ISSUE TITLE
Take a look at these lists

### ISSUE THREAD

**Issue by udit-001** [StevenBlack/hosts#523]
Found these lists which might be useful, take a look : 

List of Phishing Sites 
https://openphish.com/feed.txt

List of domains associated with Ransomware
https://ransomwaretracker.abuse.ch/downloads/RW_DOMBL.txt

**Comment by StevenBlack** [comment]
Thanks Udit @udit-001.  In particular, https://ransomwaretracker.abuse.ch looks very interesting.

**Comment by dnmTX** [comment]
@StevenBlack  i just checked some of the Phishing sites and some were blocked by the hosts file(means they're already in your list) and some by my adblocker(means they're not in the list but are legit pfishing sites). Probably it's a good idea the missing ones to be added for extra protection.

**Comment by buzzdeee** [comment]
https://ransomwaretracker.abuse.ch/downloads/RW_DOMBL.txt
# Ransomware Tracker has been discontinued on Dec 8th, 2019

that list seems to be gone

**Comment by spirillen** [comment]
WHAT?? 

danmed, back to the code :unamused:

**Comment by StevenBlack** [comment]
As for the remaining source, https://openphish.com/feed.txt, that's not a hosts file, so we would need a pre-process transformation for that.  Other issues here imply transformations, too...

**Comment by spirillen** [comment]
That's simple

```
wget -qO- 'https://openphish.com/feed.txt' | cut -d "/" -f 3 | sort -u | uniq -u
```

If you like I can add it to my https://gitlab.com/my-privacy-dns/external-sources/hosts-sources but that would be without the prefixed IP

If requested it can then be added to the https://gitlab.com/my-privacy-dns/rpz-dns-firewall-tools/hosts

@StevenBlack just say the word and it will be alive

**Comment by StevenBlack** [comment]
@spirillen yeah but what I mean is, we need an in-framework pipeline for that.

**Comment by StevenBlack** [comment]
Note to self: 1816 unique domains.

**Comment by llacb47** [comment]
By the way, it's a feed, so the URLs are updated every 12 hours. `https://openphish.com/phishing_feeds.html`

**Comment by spirillen** [comment]
```
wget -qO- 'https://openphish.com/feed.txt' | cut -d "/" -f 3 | sort -u | uniq -u | wc -l
908

wget -qO- 'https://openphish.com/feed.txt' | cut -d "/" -f 3 | wc -l
1795
```

**Comment by williesmith78** [comment]
> wget -qO- 'https://openphish.com/feed.txt' | cut -d "/" -f 3 | sort -u | uniq -u | wc -l
> 908
> 
> wget -qO- 'https://openphish.com/feed.txt' | cut -d "/" -f 3 | wc -l
> 1795
